style.css

```diff
@@ -423,3 +450,188 @@ span {
 }
 
 /* contact form end */
+
+/* Desktop Version Start */
+
+@media only screen and (min-width: 992px) {
+  .desk-nav {
+    display: block;
+    display: flex;
+  }
+
+  .desktop-image {
+    display: block;
+  }
+
+  .lang-item {
+    display: block;
+  }
+
+  .desk-nav li {
+    margin: 0 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .text-2 {
+    width: 90%;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    margin: 48px 0;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .introduction {
+    background-image: url(./Images/Header-desktop.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 168px 169px;
+    height: 815px;
+    background-size: cover;
+    width: 100%;
+  }
+
+  header {
+    padding: 28px 117px;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .contact-wrapper-two {
+    background-image: url(/Images/contact-desktop.png);
+    background-repeat: no-repeat;
+    background-position-x: right;
+    padding-top: 50px;
+    background-size: cover;
+    width: 100%;
+  }
+
+  .project-card {
+    flex-direction: row;
+    gap: 20px;
+    margin-bottom: 84px;
+    border-radius: 18px;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+  }
+
+  .project-img {
+    display: none;
+  }
+
+  .project-img img {
+    width: 100%;
+  }
+
+  .card-info {
+    width: 50%;
+    margin: 20px;
+  }
+
+  .project-btn {
+    margin: 15px 0;
+  }
+
+  .project-btn:disabled {
+    border: 2px solid grey;
+    background: white;
+    color: grey;
+    width: 140px;
+    height: 48px;
+    border-radius: 8px;
+  }
+
+  .project-btn:hover {
```

Comment:
**Kindly fix this issue:**
When you hover the button project's the button increase their size.
- Provide the requested design by the guidelines:
**_Figma template:_**
<img width="1231" alt="Screen Shot 2021-12-11 at 7 13 28" src="https://user-images.githubusercontent.com/22258274/145677861-ee0fd12e-04a3-46a0-a816-ad833aea0fc0.png">

#
**_Yours:_**
<img width="1289" alt="Screen Shot 2021-12-11 at 7 15 03" src="https://user-images.githubusercontent.com/22258274/145677961-d4519829-bdc4-44b5-8fd5-82ace5c2fa18.png">